main/windows/frames/index.ts

```diff
@@ -60,6 +60,7 @@ export default class FrameManager {
 
       if (!focusedFrame.isFocused()) {
         focusedFrame.show()
+        focusedFrame.focus()
```

Comment:
what happens if we don't have this line? docs say that `show()` shows and gives focus 
https://www.electronjs.org/docs/v14-x-y/api/browser-window#winshow

Reply:
when the window is on a different desktop it doesn't focus the window correctly -- we should also maybe reposition the window to the current desktop here